### PR TITLE
Port normalize, linear_transformation, compose, random_apply, gaussian_blur to pytest

### PR DESCRIPTION
**Pull request by vivekkumar7089**
Refactor Group E as mentioned in #3987 
- group E
   - test_normalize
   - test_linear_transformation
   - test_compose
   - test_random_apply
   - test_gaussian_blur -- parametrize over meth_kwargs

### CODE REVIEW

**Comment by NicolasHug** [comment]
Thanks @vivekkumar7089 ! from a quick look it looks like there were some merging issues: I see no deleted tested, and it seems that there are tests like `test_five_crop` etc that we recently ported in https://github.com/pytorch/vision/pull/4010

Would you mind fixing these?

**Comment by vivekkumar7089** [comment]
@NicolasHug, I will fix them out.

File: test/test_transforms_tensor.py [hunk 1]
```diff
@@ -143,132 +143,149 @@ def test_random_autocontrast(self):
...
-        s_transforms = torch.nn.Sequential(*transforms.transforms)
+            scripted_fn.save(os.path.join(tmp_dir, "t_op_list_{}.pt".format(method)))
 
-        scripted_fn = torch.jit.script(s_transforms)
-        torch.manual_seed(12)
-        transformed_tensor = transforms(tensor)
-        torch.manual_seed(12)
-        transformed_tensor_script = scripted_fn(tensor)
-        assert_equal(transformed_tensor, transformed_tensor_script, msg="{}".format(transforms))
+    def test_five_crop(self):
```
Comment (NicolasHug) [inline test/test_transforms_tensor.py:198]: I think this (and some other tests) should be left out, as they've been ported to pytest already

**Comment by vivekkumar7089** [comment]
Hi @NicolasHug !!
Now can you please review, all those tests are removed or not?

**Review by NicolasHug — APPROVED** [review]
Thanks a lot @vivekkumar7089 ! LGTM

**Comment by NicolasHug** [comment]
Good bot!
CC @pmeier ;)
